### ISSUE TITLE
[Bug] [Accessibility]:  Focus in High Contrast sometimes is not visible

### ISSUE THREAD

**Issue by jonaskuntoro** [ag-grid/ag-grid#13219]
### Link to reproducible scenario

https://www.ag-grid.com/example/

### Describe the bug

Steps to reproduce 
1. Normal contrast: 

- Go to any filter in the table, click on the fist element, and move down to select another filter and check:
- We can see in which element we are (OK)
- Go to any row in the table and check: 
- We can see in which row we are (OK)

2. High Contrast: 

- Press, at the same time, left Alt+left Shift+Printscrn in order to change to High Contrast (almost everything will be black and white)
- Go to any filter in the table, click on the first element, and move down to select another filter and check:
- We can see in which element we are (KO: I can't see in which element I am, though I can select any of them)

![Image](https://github.com/user-attachments/assets/d9d4aecc-e724-4a98-b0c4-da8128ec5b33)

- Go to any row in the table (with mouse or tab) and check: 
- We can see in which row we are (KO: all the rows look the same, though we I can select any of them by clicking with the left button of the mouse or with Space)

![Image](https://github.com/user-attachments/assets/c0ce0699-9690-4408-8290-f90c8d471651)

Actual behaviour
Sometimes the focus while on High Contrast mode is not visible, this happens on some of the Grid feature (Filter dialog, cells, headers, etc)

Expected behaviour
Focus should always be visible in both normal and High contrast mode

More information

### Version

Demo

### Does the issue occur for a specific framework only?

All languages

### Is the issue only observable on a specific browser?

All browsers

**Comment by kostasntolkeras** [comment]
Hello @jonaskuntoro , 
 
Thank you for reporting this - much appreciated!
 
Based on your description we've now reproduced this issue as a bug.
 
We have added this bug to our backlog and we are tracking it with the following reference and description:
**AG-16839 [Regression] [Accessibility] Focus & UI Indicators Not Visible in High Contrast Mode**
 
We try to fix bugs from one release to the next, so this should be fixed in the next release or the one after if it was raised too close to the next release date.
 
You can get an update on the status of this item on the product pipeline page:
https://www.ag-grid.com/ag-grid-pipeline/
 
**Now that this issue is recorded in our backlog, we will soon close the GitHub issue because we will not update it when this bug is fixed in a future version of AG Grid. The best way to get an update on this item is to sign up for AG Grid new release notifications. Once a new AG Grid version is released, look for the item reference above (AG-16839) on the changelog page to see if it's resolved in that version:
https://www.ag-grid.com/ag-grid-changelog/**
 
Thanks again for bringing this up with us - we appreciate your time!

**Comment by kostasntolkeras** [comment]
Hello @jonaskuntoro ,

Following further investigation, we’ve consolidated this work into our existing High Contrast / forced-colors accessibility items, which cover the cases reported here.

We’ll therefore close this GitHub issue and the previously referenced AG-16839 ticket as duplicates, with the work continuing under: **AG-6163 [Accessibility] Allow focused items to be clearly visible when using Windows high-contrast themes**
and
**AG-16777 [Styling] Support selection states and icons in forced-colors mode**

You can track when these items are selected for implementation on the NEXT RELEASE tab of the product pipeline:
https://www.ag-grid.com/ag-grid-pipeline/

I would recommend signing up for AG Grid new release notifications using [the instructions here](https://ag-grid.zendesk.com/hc/en-us/articles/360002648058-How-do-I-sign-up-for-ag-Grid-release-notifications-). This way you'll know as soon as a new version is out and you can check whether this was implemented on [the changelog page](https://www.ag-grid.com/ag-grid-changelog/).

Thanks again for reporting this!
